src/main/scala/nl.knaw.dans.easy.validatebag/XmlValidator.scala

```diff
@@ -27,12 +27,13 @@ import scala.util.Try
 
 class XmlValidator(schema: Schema) {
 
+  val validator = schema.newValidator()
```

Comment:
If the validator can be used over and over again, create it in a companion object. Thus it becomes a static instance. Now you are only creating an instance that is used if validate is not called.